app/src/js/components/Collections/list.js

```diff
@@ -89,6 +90,10 @@ class CollectionList extends React.Component {
     const { count, queriedAt } = list.meta;
     return (
       <div className="page__component">
+        <Helmet>
+          <meta charset= "utf-8" />
+          <title> Collection List </title>
```

Comment:
We can just have this say "Collections"
```suggestion
          <title> Collections </title>
```